Sitka-Sound-Science-Center/sitka-landslide
# Frontend application

## Overview

Adds Next.js frontend application.

### Demo

<img width="1395" alt="Screen Shot 2022-03-22 at 2 56 34 PM" src="https://user-images.githubusercontent.com/1809908/159555656-3ab31a51-364f-43bc-843d-a05db9efe30e.png">

### Notes

- The Next.js build process seems slow on Netlify—we might consider using Vercel, which appears to build the site 3–4x faster, which will have an impact on pricing
- I need to write additional CSS to match the designs for desktop (#10)

## Testing Instructions

 * Make sure to run ./scripts/update
 * Run the application
 * Click on everything! The hamburger menu in the header, the accordions under "3 day forecast", every link
 * If you modify `/frontend/data/index.json` you should see the changed reflected on the homepage

## Checklist
